x-pack/plugins/alerting/server/types.ts

```diff
@@ -29,6 +36,8 @@ declare module 'src/core/server' {
 }
 
 export interface Services {
+  // This will have to remain `any` until we can extend Alert Services with generics
```

Comment:
Is this comment correct?  I thought the `any`s for callCluster was because callCluster definition, not because of our own generics.

Reply:
Nope, mistake, good catch :)

Reply:
Actually... looking at it I've remembered why I marked it that way - this is *our* exposed interface to the plugins and we *could* exposed it as `unknown` if we wanted to. Exposing as `any` allows plugins to avoid explicitly casting, which means they're more likely to drag a lack of typechecking into their code (`any` is pervasive, if they even reference this response (in the `Promise<any>`) then anything referenced from it will not be type checked, even though it might look like it is.

If we expose as `unknown` they would have to explicitly state what type they think it is and it will typecheck properly from there out.

But changing that interface isn't obvious and we'd have to look into it further.